<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0" rowspan="4">
     <widget class="QFrame" name="OptionsFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QFrame" name="Options1">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Registering and Stacking</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeView" name="OptionsTree1"/>
          </item>
          <item>
           <widget class="QTreeView" name="OptionsTree2"/>
          </item>
          <item>
           <widget class="QTreeView" name="OptionsTree3"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="Options2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Processing</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeView" name="OptionsTree4"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="Options3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Options</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeView" name="OptionsTree5"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="ControlFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QFrame" name="GraphicsFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>3</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_New_project"/>
    <addaction name="action_Load_project"/>
    <addaction name="action_Save_project"/>
    <addaction name="action_Clear_file_list"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Registering</string>
    </property>
    <addaction name="action_Dark_frames"/>
    <addaction name="action_Flat_frames"/>
    <addaction name="actionDar_k_Flat_frames"/>
    <addaction name="action_Bias_frames"/>
   </widget>
   <widget class="QMenu" name="menuStacking">
    <property name="title">
     <string>Stacking</string>
    </property>
    <addaction name="actionCreate_star_mask"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_DSS"/>
    <addaction name="actionHelp_topics"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuProcessing_2">
    <property name="title">
     <string>Processing</string>
    </property>
    <addaction name="actionOpen_picture_file"/>
    <addaction name="actionCreate_star_mask_2"/>
    <addaction name="actionCopy_current_picture_to_clipboard"/>
    <addaction name="actionSave_picture_file"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuStacking"/>
   <addaction name="menuProcessing_2"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Load_project">
   <property name="text">
    <string>&amp;Load file list</string>
   </property>
  </action>
  <action name="action_Save_project">
   <property name="text">
    <string>&amp;Save file list</string>
   </property>
  </action>
  <action name="action_New_project">
   <property name="text">
    <string>&amp;New file list</string>
   </property>
  </action>
  <action name="action_Dark_frames">
   <property name="text">
    <string>&amp;Dark frames</string>
   </property>
  </action>
  <action name="action_Flat_frames">
   <property name="text">
    <string>&amp;Flat frames</string>
   </property>
  </action>
  <action name="actionDar_k_Flat_frames">
   <property name="text">
    <string>Dar&amp;k Flat frames</string>
   </property>
  </action>
  <action name="action_Bias_frames">
   <property name="text">
    <string>&amp;Bias frames</string>
   </property>
  </action>
  <action name="actionCreate_star_mask">
   <property name="text">
    <string>Compute offset</string>
   </property>
  </action>
  <action name="actionAbout_DSS">
   <property name="text">
    <string>About DSS</string>
   </property>
  </action>
  <action name="action_Clear_file_list">
   <property name="text">
    <string>&amp;Clear file list</string>
   </property>
  </action>
  <action name="actionOpen_picture_file">
   <property name="text">
    <string>Open picture file</string>
   </property>
  </action>
  <action name="actionSave_picture_file">
   <property name="text">
    <string>Save picture file</string>
   </property>
  </action>
  <action name="actionCreate_star_mask_2">
   <property name="text">
    <string>Create star mask</string>
   </property>
  </action>
  <action name="actionCopy_current_picture_to_clipboard">
   <property name="text">
    <string>Copy current picture to clipboard</string>
   </property>
  </action>
  <action name="actionHelp_topics">
   <property name="text">
    <string>Help topics</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
